Public/Src/Engine/Scheduler/ProcessRunnablePip.cs

```diff
@@ -21,6 +21,16 @@ public sealed class ProcessRunnablePip : RunnablePip
         /// <nodoc/>
         public Process Process => (Process)Pip;
 
+       
+        /// <summary>
+        /// Process weight
+        /// </summary>
+        /// <remarks>
+        /// If process weight is defined as greater than 1 in the specs, use it. 
+        /// Otherwise, use the weight based on historic cpu usage.
+        /// </remarks>
+        public int Weight => Process.Weight > 1 ? Process.Weight : m_weightBasedOnHistoricCpuUsage;

```

Comment:
> 1 [](start = 45, length = 2)

Can you put this in a "MIN_PROCESS_WEIGHT" or similar const somewhere that is also shared with the definition of Weight in Process.cs?

Reply:
Not going to block PR on this, but we should definitely do it :) 

---
In reply to: [277380578](https://github.com/Microsoft/BuildXL/pull/178#discussion_r277380578) [](ancestors = 277380578)